src/tests/align.test.tsx

```diff
@@ -63,517 +54,531 @@ const createAndSelectTwoRectanglesWithDifferentSizes = () => {
   });
 };
 
-it("aligns two objects correctly to the top", () => {
-  createAndSelectTwoRectangles();
-
-  expect(API.getSelectedElements()[0].x).toEqual(0);
-  expect(API.getSelectedElements()[1].x).toEqual(110);
-
-  expect(API.getSelectedElements()[0].y).toEqual(0);
-  expect(API.getSelectedElements()[1].y).toEqual(110);
-
-  Keyboard.withModifierKeys({ ctrl: true, shift: true }, () => {
-    Keyboard.keyPress(KEYS.ARROW_UP);
+describe("aligning", () => {
+  beforeEach(async () => {
+    // Unmount ReactDOM from root
+    ReactDOM.unmountComponentAtNode(document.getElementById("root")!);
+    mouse.reset();
+
+    await setLanguage("en.json");
+    await render(<AppWithCollab />);
+    // FIXME unless there's a solid wait after render we get some suepr weird
+    // flake
+    await new Promise((r) => setTimeout(r, 300));
```

Comment:
No idea what this is about. If we don't wait some time after render, there's some state (`elements` in particular don't get cleared) leaking into next tests, breaking them.